StripeConnect/StripeConnect/Source/Internal/Webview/ConnectWebView.swift

```diff
@@ -111,16 +110,31 @@ private extension ConnectWebView {
         urlOpener.openIfPossible(url)
     }
 
-    func showErrorAlert(for error: Error) {
+    func showErrorAlert(for error: Error?) {
         // TODO: MXMOBILE-2491 Log analytic when receiving an eror
-        debugPrint(error)
+        debugPrint(String(describing: error))
 
         let alert = UIAlertController(
             title: nil,
             message: NSError.stp_unexpectedErrorMessage(),
             preferredStyle: .alert)
         presentPopup(alert)
     }
+
+    func cleanupDownloadedFile() {
+        guard let downloadedFile else { return }
+
+        // Delete file since we don't need it anymore
+        DispatchQueue.global(qos: .background).async { [weak self] in
+            try? self?.fileManager.removeItem(at: downloadedFile)
```

Comment:
Oh, I remember why I didn't do that now. I'm not well versed enough in Swift 6 language mode issues to know what the right fix is for this, so lmk if you have any suggestions. Otherwise I think the best option is to leave it as-is.

<img width="779" alt="image" src="https://github.com/user-attachments/assets/546e33ae-057d-4b6d-861d-263b0446bf86">